content/_index.md

```diff
@@ -1,50 +1,33 @@
 ---
-title: "London Gophers"
+title: "Welcome to London, Gopher!"
 date: 2019-05-09T15:28:25+01:00
 ---
 
 London Gophers, aka The Go London User Group (GLUG), are a London-based community for anyone interested in the Go
 programming language. London Gophers held their first Meetup on Wednesday, March 27, 2013, and since then the group has
 grown to over 3,000 members.
 
-London Gophers typically hold a Meetup once a month at a variety of venues across the city. All Meetups are free, and
-typically comprise a number of talks and socialising, with food and drink provided. We live-stream all Meetups via our
-[YouTube channel](https://youtube.com/LondonGophers), where you can also find an archive of previous recordings.
-
-Membership of London Gophers is managed via our [Meetup page](https://www.meetup.com/LondonGophers/); everyone is
-welcome! We also have an active [Twitter feed](https://twitter.com/LondonGophers) and our members are active in the
-`#london` [Gophers Slack](https://gophers.slack.com/) ([sign-up here](https://invite.slack.golangbridge.org)).
-
-### Code of Conduct
-
-London Gophers operate under the [Go Community Code of Conduct](https://golang.org/conduct).
-
 ### Contact
 
 For any questions about London Gophers, our Meetups, speaking at one of our events, sponsoring one of our events,
-hosting one of our events, please get in touch with [the organisers](mailto:glug-organisers@googlegroups.com).
+hosting one of our events, you can contact us at [contact@gophers.london](mailto:contact@gophers.london).
 
-### Sponsors
+### Upcoming Events
 
-London Gophers is a volunteer-run organisation that seeks to remain as friendly and welcoming as possible to everyone.
-Therefore membership of London Gophers is 100% free and we do not charge for attending events. Not only that, but we
-provide food and drink at all of our events. We therefore rely on sponsorship to cover our costs. Thank you to our
-sponsors!
+London Gophers typically hold a Meetup once a month at a variety of venues across the city. All Meetups are free, 
+and consists of a number of talks and socialising, with food and drink provided.
 
-<div class="sponsors">
-<a href="https://www.utilitywarehouse.co.uk" target="_blank"><img alt="Utility Warehouse logo" src="/images/sponsors/utilitywarehouse.jpg"/></a>
-<a href="https://tyk.io/" target="_blank"><img alt="Tyk logo" src="/images/sponsors/tyk.png"/></a>
-<a href="https://search.gocenter.io/" target="_blank"><img alt="JFrog GoCenter logo" src="/images/sponsors/gocenter.png"/></a>
-</div>
+{{< article link="/meetups/slot_1/" >}}
+{{< article link="/meetups/slot_2/" >}}
+{{< article link="/meetups/slot_3/" >}}
```

Comment:
I guess these can be archetypes and they can be generated automatically based on the archetype/category. We can leave as it is for now and investigate how this works later on.

This way, we only need to add a "post" and it automatically will add the articles here.

Reply:
If that's possible it makes sense, I've not looked into 'archetypes' so I don't know.